### ISSUE TITLE
Improve the "Data Structures and Containers" section

### ISSUE THREAD

**Issue by choldgraf** [mne-tools/mne-python#4188]
This is for once the docs refactor is merged. We should add a section for each of the major data structures. Things I can think of that are missing are:

- [ ] AverageTFR
- [ ] EpochsTFR
- [ ] STC objects
- [ ] Annotations
- [ ] events (I know it's just a numpy array but everyone keeps asking about this)

**Comment by jona-sassenhagen** [comment]
events + event_id!

**Comment by agramfort** [comment]
@drammock do you think we can close this one?

**Comment by drammock** [comment]
I think so, because I think it referred to the old "landing page" of the docs. No longer relevant.
